README.md

```diff
@@ -0,0 +1,161 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="cat-black-face.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio: setup and mobile first</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+Hello Microverse is a "Hello world" repository exercise. The goal here is to master all of the tools and best practices learned about in previous steps to use them in all Microverse projects and most likely in my future job as well.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML 5</a></li>
+  </ul>
+  <ul>
+    <li>CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Use of linters**
+- **Showcase of branching and pull request**
+- **Example of profesional best practices**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. Git
+2. A Web Browser with developer tools that supports HTML5 and CSS.
+
+### Setup
+1. On GitHub.com, navigate to the main page of the repository.
+2. Above the list of files, click *Code*.
+3. Copy the URL for the repository.
+4. Open Terminal.
+5. Change the current working directory to the location where you want the cloned directory.
+6. Type git clone, and then paste the URL you of the reposotory.<br>
+``` $ git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY ```
```

Comment:
- [ ] Kindly make sure to remove this placeholder and add your repository for cloning purpose.

Reply:
Thank you very much and made the changes